libvirt/libvirt_test.go

```diff
@@ -96,12 +107,82 @@ func TestLibirt(t *testing.T) {
 		expectedType := make(map[string]int64)
 		So(cpu, ShouldHaveSameTypeAs, expectedType)
 	})
+	Convey("Get Network Statistics without network interface neither path", t, func() {
+		conn, err := libvirt.NewVirConnection("test:///default")
+		defer conn.CloseConnection()
+		So(err, ShouldBeNil)
+		domains, err := GetInstances(conn)
+		network, err := GetNetworkStatistics(domains[0])
+		So(err, ShouldNotBeNil)
+		emptyMap := make(map[string]libvirt.VirDomainInterfaceStats)
+		So(network, ShouldResemble, emptyMap)
```

Comment:
I suggest checking values returned by functions, not just checking if type is correct. This is general comment, please take a look at the rest of tests which you added and consider it.